ui/src/app/service/utils.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+/**
+ * Common methods to be used cross all components
+ */
+export class UtilsService {
```

Comment:
This seems to be a stateless class. You can ignore the class and just export functions. (And you don't need to inject it.)